Add note about binding "," to invoke major mode if it isn't bound (#19)
VSpaceCode/vspacecode.github.io: docs/major-mode.md

Major mode offers dynamic menu and functions based on the active buffer. You can access key bindings specific to the current major mode by pressing `<spc> m` or `,` if you want to save one keystroke.

:::note

> If `,` isn't bound, you might need to update your settings. You can do this by running the command "VSpaceCode: Configure Default Settings" (this will add the necessary keybindings to your `settings.json`). More information can be found in the [Installation](./) instructions.

Reviewer: ```suggestion
If `,` isn't bound, you might need to update your settings. You can add the necessary keybindings to your `settings.json` by pressing `Ctrl+Shift+p` and running the command "VSpaceCode: Configure Default Settings". More information can be found in the [Installation](./) instructions.
```
